<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DuelSC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="HostAdd">
       <property name="text">
        <string>+1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HostSubtract">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectorAdd">
       <property name="text">
        <string>+1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectorSubtract">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="PlayerLabel">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="HostScore">
       <property name="text">
        <string>Score: 25</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="HostDiscount">
       <property name="text">
        <string>Discounts: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>81</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="PlayerLabel2">
       <property name="text">
        <string>Connector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ConnectorScore">
       <property name="text">
        <string>Score: 25</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ConnectorDiscount">
       <property name="text">
        <string>Discounts: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
